<launch>
	<!--arg name="marker_size" default="3.3" /-->
	<arg name="marker_size" default="13.3" />
	<arg name="max_new_marker_error" default="0.05" />
	<arg name="max_track_error" default="0.2" />

	<arg name="cam_image_topic" default="/head_mount_kinect/depth_registered/points" />
	<arg name="cam_info_topic" default="/head_mount_kinect/rgb/camera_info" /> 
	<!--<arg name="cam_image_topic" default="/camera/depth_registered/points" /> --> 
	<!--<arg name="cam_info_topic" default="/camera/rgb/camera_info" /> -->
	<arg name="output_frame" default="/torso_lift_link" />        

	<!-- Vision sensor -->
	<node name="ar_track_alvar" pkg="ar_track_alvar" type="individualMarkers" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame)" />

	<!-- Find AR Tag on Autobed -->
	<node name="find_autobed" pkg="hrl_base_selection" type="find_autobed.py" respawn="false" output="screen" />

	<!-- Find AR Tag on Autobed -->
	<node name="find_head" pkg="hrl_base_selection" type="find_head.py" respawn="false" output="screen" />

</launch>
